### PR TITLE
Onboard  GoogleCloudPlatform/blueprints repo

### PR DESCRIPTION
**Pull request by bharathkkb**
fixes https://github.com/GoogleCloudPlatform/blueprints/issues/6

### CODE REVIEW

**Comment by bharathkkb** [comment]
Assigning to oncall
/assign @cjwagner

**Comment by bharathkkb** [comment]
For enabling in repo configs should I also add repo here?
https://github.com/GoogleCloudPlatform/oss-test-infra/blob/c3f7b083043e358835fe2a48bd929fb5fd8e158c/prow/oss/config.yaml#L222

**Comment by cjwagner** [comment]
> For enabling in repo configs should I also add repo here?
> https://github.com/GoogleCloudPlatform/oss-test-infra/blob/c3f7b083043e358835fe2a48bd929fb5fd8e158c/prow/oss/config.yaml#L222

Yes. You'll also need to indicate the build clusters that jobs defined in your repo are allowed to schedule to in the config  field below that. This should be the build cluster alias produced from the first onboarding step: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/oss/onboarding.md#set-up-build-cluster

You can also enable the trigger plugin now as described in the onboarding guide: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/oss/onboarding.md#config-prow-to-act-on-webhooks-from-github

File: prow/oss/config.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ tide:
     GoogleCloudPlatform/osconfig: squash
     GoogleCloudPlatform/artifact-registry-apt-transport: squash
     GoogleCloudPlatform/artifact-registry-yum-plugin: squash
+    GoogleCloudPlatform/blueprints: squash
```
Comment (cjwagner) [inline prow/oss/config.yaml:26]: FYI this doesn't actually do anything unless you enable the merge automation by modifying the queries below. Thats for the best though. You shouldn't add merge automation until you have presubmits set up.

**Review by cjwagner — APPROVED** [review]

**Pull request merged**